tests/bridge/integration/peg_out/utils.rs

```diff
@@ -67,7 +67,9 @@ pub async fn create_and_mine_kick_off_2_tx(
         &operator_context.n_of_n_taproot_public_key,
         &HashMap::from([(
             CommitmentMessageId::Superblock,
-            WinternitzPublicKey::from(&commitment_secrets[&CommitmentMessageId::Superblock]),
+            WinternitzPublicKeyVariant::Standard(WinternitzPublicKey::from(
+                WinternitzPublicKey::from(&commitment_secrets[&CommitmentMessageId::Superblock]),
+            )),
```

Comment:
Do we need a double `WinternitzPublicKey` conversion?

Reply:
Well spotted! A copy pasta.

Reply:
DONE